<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="Input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Output">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>371</width>
     <height>281</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Add">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FontUbuntu">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubuntu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FontNanumGothic">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>NanumGothic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Red">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Green">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>220</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Blue">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Clear">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetFontSize">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Font Size</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FontSize">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
